src/components/forums/Forums.js

```diff
@@ -120,30 +142,71 @@ const Forums = () => {
                     onButtonClick={loadGetForums}
                     onResetSearch={handleRefreshSearch}
                 />
+
+                <br />
+                <Typography
+                    style={{ color: "black", mb: 2, fontSize: 14, align: 'right' }}
+                >
+                    Filters
+                </Typography>
+                <DropdownFilter
+                    placeholder="Select a Forum Tag"
```

Comment:
Is it possible to make the placeholder text gray?

Reply:
I can style once merged